test/helpers/readAsset.js

```diff
@@ -1,7 +1,7 @@
 import path from "path";
 
-export default (asset, compiler, stats) => {
-  const usedFs = compiler.outputFileSystem;
+export default (asset, stats) => {
```

Comment:
Avoid these changes, it is auto generated utils

Reply:
Where are they generated from?

Reply:
From our boilerplace

Reply:
b6c480a2849b9a09c5769ef5d313af5741cbe4ff